pkg/services/cloudmigration/api/api.go

```diff
@@ -89,6 +91,16 @@ func (cma *CloudMigrationAPI) GetToken(c *contextmodel.ReqContext) response.Resp
 	})
 }
 
+// swagger:parameters getCloudMigrationToken
+type GetCloudMigrationToken struct {
+}
+
+// swagger:response cloudMigrationGetTokenResponse
+type CloudMigrationGetTokenResponse struct {
+	// in: body
+	Body cloudmigration.GetAccessTokenResponseDTO
```

Comment:
You don't need to change it now if you don't want to but I think that for separation of concerns, the best thing we could do is to:

- Have a viewmodel, which are the structs used to unmarshal the data received the requests and the marshal the data that will be in the responses
- Have the domain model, which are the structs that represent the things in our domain
- Do not expose the domain model in the responses aka don't use the domain structs in the http responses (domain model can change, the public api can't)

Reply:
Done! PTAL